docs/marketplace/dev/feature-integration-guides/202108.0/glue/marketplace-product-feature-integration.md

```diff
@@ -135,14 +110,17 @@ class GlueApplicationDependencyProvider extends SprykerGlueApplicationDependency
 }
 ```
 
-</details>
-
 {% info_block warningBox "Verification" %}
 
-Make sure that you can add a merchant product to the cart using a `POST` request to `http://glue.de.demo-spryker.com/guest-cart-items or http://glue.de.demo-spryker.com/carts/{% raw %}{{idCart}}{% endraw %}/items`.
-
-Make sure that when you do a `GET` request for the carts with merchant products, their merchants are returned as well. `http://glue.de.demo-spryker.com/guest-carts/{idCart}?include=guest-cart-items,merchants` or `http://glue.de.demo-spryker.com/carts/{% raw %}{{idCart}}{% endraw %}?include=items,merchants`.
-
 Make sure that when you do a `GET` request to retrieve abstract products that belong to a specific merchant, it returns products' data together with their merchants `http://glue.de.demo-spryker.com/abstract-products/{% raw %}{{abstractProductSku}}{% endraw %}?include=merchants`.
 
 {% endinfo_block %}
+
+
+## Related features
+
+Integrate the following related features:
+
+| FEATURE | REQUIRED FOR THE CURRENT FEATURE |INTEGRATION GUIDE |
+| --- | --- | --- |
+| Glue API: Marketplace Product + Cart |  |[Glue API: Marketplace Product + Cart feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/glue/marketplace-product-cart-feature-integration.html) |
```

Comment:
An optional IG like Glue API: Marketplace Product + Cart should have a link to the main IG. Can be removed and added in Glue API: Marketplace Product + Cart.

Reply:
@ilyakubanov
do you mean that no need link on `Glue API` level, instead all tied up in main top feature like this:
```
Marketplace Product Feature related:
    Marketplace Product + Cart Feature
    Glue API: Marketplace Product + Cart Feature
```

Reply:
removed and added link on main `Marketplace Product + Cart feature integration`